.github/workflows/doc_build.yml

```diff
@@ -1,124 +0,0 @@
-name: Docs
```

Comment:
Isn't that job potentially useful to check the output of a pull request touching docs, thanks to the artifacts ? (unless there's an equivalent with RTD)

Reply:
The RTD check should fail in a similar way, but I'll check in this PR with a dummy commit (soon).

Reply:
ah, I was thinking more on downloading the artifacts to check they look nice, but I see that the RTD job has an even nicer output that can be directly visualized, so my point is moot